test/testFunctions.js

```diff
@@ -86,7 +86,6 @@
       animation.currentTime = at;
       var result = target.style._getAnimated(propertyToRead);
       animation.cancel();
-
       assert.equal(result, is, 'For: ' + JSON.stringify(keyframes) + ' at: ' + at + '\n');
```

Comment:
You should put the blank line back as there isn't a good reason for this patch to change this file.